x/emissions/keeper/inference_synthesis/network_losses.go

```diff
@@ -210,43 +210,6 @@ func CalcNetworkLosses(
 	return output, nil
 }
 
-// Same as CalcNetworkLosses() but just returns the combined loss
-func CalcCombinedNetworkLoss(
```

Comment:
is there a reason we're deleting this test, don't love deleting tests without a reason

Reply:
this isn't a test though right?

Reply:
oh for some reason i thought it was in network_losses_test.go